lookout/core/analyzer.py

```diff
@@ -68,7 +68,7 @@ class Analyzer:
     model_type = None  # type: Type[AnalyzerModel]
     name = None  # type: str
 
-    def __init__(self, model: AnalyzerModel, url: str, config: Mapping[str, Any]):
+    def __init__(self, model: AnalyzerModel, url: str, config: Mapping[str, Mapping[str, Any]]):
```

Comment:
One question I have, how can we be sure that it is the same config we use for training?

Reply:
ok, not the same, but all training related stuff is the same.

Reply:
we're not sure, that's why we have to save the config in modelforge for the parts that need to be the same for the model to work.

Reply:
oh, sure!